apps/web/lib/web/application.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Web.Application do
+  # See https://hexdocs.pm/elixir/Application.html
+  # for more information on OTP Applications
+  @moduledoc false
+
+  use Application
+
+  def start(_type, _args) do
```

Comment:
Does this need to be a supervised application or is it just a library?

Reply:
It's is just a library

Reply:
It doesn't need this application file then. See the Providers library in apps.

Reply:
I've deleted the application.ex file